src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py

```diff
@@ -1000,11 +1009,18 @@ def calculate_jvm_modulizable_targets(self):
         def is_jvm_or_resource_target(t):
             return isinstance(t, (JvmTarget, JvmApp, JarLibrary, Resources))
```

Comment:
We may want to add `target` and `alias` targets here, or dealias somehow, otherwise these will be filtered out.

Reply:
SOmething like
```

        def is_jvm_or_resource_target(t):
            return isinstance(t, (JvmTarget, JvmApp, JarLibrary, Resources)) or t.type_alias == 'target'
```

Reply:
This change fixes the issue I have with `./pants export-dep-as-jar util/util-slf4j-api/::`, where the target-dependencies of `util/util-slf4j-api/src/test/java/com/twitter/util/logging:logging` are missing.

Reply:
Thanks. Adjusted.